app/views/boards/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<body>
+
+ <div class="container">
+
+  <div class="board">
+
+    <table class="one">
+      <% 4.times do %>
+     
+        <tr class="even">
+ 	       <% 8.times do %>
+ 	        <td></td>	 
+ 	       <% end %>
+ 	      </tr>
+  
+        <tr class="odd">
+         <% 8.times do %>
+ 	        <td></td>
+         <% end %>
+ 			  </tr>
+ 			
+      <% end %>
+
+	  </table>
+  </div>
+ </div>
```

Comment:
Ya, it's really annoying that it happened again. Much easier now with they syntax override, but obviously it's still off.